stockPrice

cjortiz287 opened the pull request.

cjortiz287 committed:
stockPrice

houyhourung commented:
what have you change in the codes? and what feature you're working on ? please specify

cjortiz287 committed:
Update cjortiz287

Added st.metric(label=polyresponse2, value=float, delta=float, delta_color="normal") that will show an increase in price as a green arrow and a decrease of stock as a red arrow.

cjortiz287 reviewed (commented):
added st.metric(label=polyresponse2, value=float, delta=float, delta_color="normal") that shows as green if the pice has increased or red if the price as decreased. Also got rid of the original code since it appeared that the price class was already coded. So just added the metric to that.

houyhourung commented on cjortiz287:
```diff
@@ -0,0 +1,49 @@
+                openprice = polyresponse2["results"][0]["o"]
+                ntrans = polyresponse2["results"][0]["n"]
+                volume = polyresponse2["results"][0]["v"]
+                st.metric(label=polyresponse2, value=float, delta=float, delta_color="normal")
```
this line is not working properly

houyhourung replied:
"TypeError: '<class 'float'>' is of type <class 'type'>, which is not an accepted type. value only accepts: int, float, str, or None. Please convert the value to an accepted type." this is the error when i run the app

houyhourung commented on cjortiz287:
```diff
@@ -0,0 +1,49 @@
+                    st.write(name + "'s Closing Price")
+                    st.write(closeprice)
+
+                    col1.metric(polyresponse2, float, float, "normal")
```
same error:
TypeError: '<class 'float'>' is of type <class 'type'>, which is not an accepted type. value only accepts: int, float, str, or None. Please convert the value to an accepted type.

houyhourung commented on cjortiz287:
```diff
@@ -0,0 +1,49 @@
+                    st.write(name + "'s Highest Price")
+                    st.write(highestprice)
+
+                    col2.metric(polyresponse2, float, float, "normal")
```
i'm assuming this is not working too

houyhourung reviewed (changes requested):
Also please make sure to pull from main and put your codes up. It seems like you're using a new file and Github tracked that so please be sure to pull from this repository in order to get the latest update.
